quarkus/runtime/src/test/java/org/keycloak/quarkus/runtime/configuration/test/LoggingConfigurationTest.java

```diff
@@ -287,6 +299,22 @@ public void testWildcardOptionFromConfigFile() {
     @Test
     public void testWildcardPropertiesDontMatchEnvVarsFormat() {
```

Comment:
I noticed that this test might be wrong, it's supposed to test env vars  but it's not doing that. But that's not related to the changes in this PR.

Reply:
Agreed, just removed the test.